packages/esm-patient-attachments-app/src/camera-media-uploader/capture-photo.component.tsx

```diff
@@ -58,7 +57,7 @@ const CapturePhoto: React.FC<CapturePhotoProps> = ({ initialState, onCapturePhot
           className={styles.editButton}
           kind="ghost"
           onClick={showCam}
-          renderIcon={(props) => <Edit {...props} />}
+          renderIcon={(props: Partial<Parameters<typeof EditIcon>[0]>) => <EditIcon {...props} />}
```

Comment:
Comparing apples to apples, but we could maybe this signature could be simplified to:

```suggestion
          renderIcon={(props: React.ComponentProps<typeof EditIcon>) => <EditIcon {...props} />}
```

Reply:
Actually, no. That's way more explicit, even if not necessarily shorter. Let me fix that.